ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/SandboxOnXForTest.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.sandbox
+
+import com.daml.ledger.api.auth.{AuthService, AuthServiceWildcard}
+import com.daml.ledger.configuration.Configuration
+import com.daml.ledger.runner.common.MetricsConfig.MetricRegistryType
+import com.daml.ledger.runner.common.{
+  Config,
+  MetricsConfig,
+  ParticipantConfig => _ParticipantConfig,
+}
+import com.daml.lf.data.Ref
+import com.daml.lf.engine.{EngineConfig => _EngineConfig}
```

Comment:
Why do we need the renaming.

Reply:
it clashes then with `val EngineConfig = EngineConfig(...` which is exposed then outside, did not want to rename that

Reply:
make sens.